From 3c024129db3fd8f312baacc0932f6f4c165a1ac2 Mon Sep 17 00:00:00 2001
From: Alexander Graf <agraf@suse.de>
Date: Sat, 19 Jan 2013 11:26:56 +0100
Subject: [PATCH 2/2] OMAP: Fix missing usb.h include
Patch-Mainline: No, too hacky

Without this, we get the following build error:

drivers/usb/musb/musb_dsps.c: In function 'musb_dsps_phy_control':
drivers/usb/musb/musb_dsps.c:156:18: error: 'USBPHY_CM_PWRDN' undeclared (first use in this function)
drivers/usb/musb/musb_dsps.c:156:18: note: each undeclared identifier is reported only once for each function it appears in
drivers/usb/musb/musb_dsps.c:156:36: error: 'USBPHY_OTG_PWRDN' undeclared (first use in this function)
drivers/usb/musb/musb_dsps.c:157:16: error: 'USBPHY_OTGVDET_EN' undeclared (first use in this function)
drivers/usb/musb/musb_dsps.c:157:36: error: 'USBPHY_OTGSESSEND_EN' undeclared (first use in this function)

Signed-off-by: Alexander Graf <agraf@suse.de>
---
 drivers/usb/musb/musb_dsps.c           | 1 +
 include/linux/platform_data/usb-omap.h | 5 +++++
 2 files changed, 6 insertions(+)

diff --git a/drivers/usb/musb/musb_dsps.c b/drivers/usb/musb/musb_dsps.c
index e6f2ae8..27e66e8 100644
--- a/drivers/usb/musb/musb_dsps.c
+++ b/drivers/usb/musb/musb_dsps.c
@@ -45,6 +45,7 @@
 #include <linux/debugfs.h>
 
 #include "musb_core.h"
+#include "../../../arch/arm/mach-omap2/usb.h"
 
 static const struct of_device_id musb_dsps_of_match[];
 
diff --git a/include/linux/platform_data/usb-omap.h b/include/linux/platform_data/usb-omap.h
index ef65b67..0da5930 100644
--- a/include/linux/platform_data/usb-omap.h
+++ b/include/linux/platform_data/usb-omap.h
@@ -19,6 +19,9 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
+#ifndef _PLATFORM_USB_OMAP_
+#define _PLATFORM_USB_OMAP_
+
 #define OMAP3_HS_USB_PORTS	3
 
 enum usbhs_omap_port_mode {
@@ -81,3 +84,5 @@ enum musb_interface {
 	MUSB_INTERFACE_ULPI,
 	MUSB_INTERFACE_UTMI
 };
+
+#endif
-- 
1.7.12.4

